GoogleForCreators/web-stories-wp
# Delete a non-draft story from storage after saving

## Context



## Summary
Fixes a bug where a story is not deleted from the session storage when saving the story.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a new story
2. Make some changes (don't save!)
3. Wait until an auto-save is created for the draft to the storage session.
4. Save now
5. Verify the local auto-save gets deleted.
6. Now make new changes (don't save!)
7. Wait until an auto-save is created for the saved story to the storage session.
8. Save the story.
9. Verify the local auto-save gets deleted.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12444
